src/WebJobs.Script.WebHost/StandbyManager.cs

```diff
@@ -44,12 +44,6 @@ public static bool IsWarmUpRequest(HttpRequestMessage request)
                 (request.MatchRoute($"api/{WarmUpFunctionName}") || request.MatchRoute($"api/{WarmUpAlternateRoute}"));
         }
 
-        public static async Task WarmUp(ScriptHost host)
-        {
-            // exercise the Node pipeline
-            await NodeFunctionInvoker.InitializeAsync();
```

Comment:
Why deleted? We want to be able to warm up the various language workers as required.